target/linux/ath79/dts/qca9561_tplink_tl-wdr7800-v1.dts

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9561_tplink_tl-wdr7800.dtsi"
+
+/ {
+  compatible = "tplink,tl-wdr7800-v1", "qca,qca9561";
+  model = "TP-Link TL-WDR7800 Version 1";
+};
+
+&mtdparts {
+  uboot: u-boot@0 {
+    reg = <0x000000 0x010000>;
+    read-only;
+  };
+
+  firmware@10000 {
```

Comment:
Please add `compatible`, see
[OpenWrt-Devel] Specifying "firmware" partition format on all DTS targets
http://lists.infradead.org/pipermail/openwrt-devel/2018-November/014706.html
for details.